functions/openhab.bash

```diff
@@ -68,8 +68,7 @@ Check the \"openHAB Release Notes\" and the official announcements to learn abou
   openhabVersion="$(apt-cache madison openhab2 | head -n 1 | cut -d'|' -f2 | xargs)"
 
   if ! cond_redirect apt-get -y --allow-downgrades install "openhab2=${openhabVersion}"; then echo "FAILED (apt)"; return 1; fi
-  cond_redirect adduser openhab gpio
```

Comment:
Why remove?

Reply:
it fails because it's done in some other place before